miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -750,16 +754,6 @@ protected LibraryDto packageDtoFromParent(Sample sample) {
       }
       dto.setParentSampleId(sample.getId());
       dto.setParentSampleAlias(sample.getAlias());
-      if (namingScheme.hasLibraryAliasGenerator()) {
-        try {
-        Library tempLibrary = new LibraryImpl();
-        tempLibrary.setSample(sample);
-        final String libraryAlias = namingScheme.generateLibraryAlias(tempLibrary);
-        dto.setAlias(libraryAlias);
-        } catch (MisoNamingException e) {
-          log.error("Failed to generate library name", e);
-        }
-      }
```

Comment:
I do need to update the DefaultLibraryAliasGenerator for the change though, so will do that